### ISSUE TITLE
An object could not be cloned.

### ISSUE THREAD

**Issue by c-h-** [chikeichan/redux-worker#12]
Cool project!!

Any idea on what this error might mean?

Thanks!

<img width="836" alt="screen shot 2016-07-25 at 2 28 42 am" src="https://cloud.githubusercontent.com/assets/5919164/17092295/9b62c1c4-520f-11e6-9b85-d48537680d6d.png">

**Comment by marchaos** [comment]
I'm seeing the same error. I suspect that it's because there is an object that contains a function, which cannot be cloned using structured cloning. https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm

See http://stackoverflow.com/questions/7704323/passing-objects-to-a-web-worker

**Comment by chikeichan** [comment]
Yep. It seems that some unsupported datatype being passed to web worker.
